microsoft/vscode
# Fixed Zen Mode not exiting when leaving fullscreen - #231474

I propose to remove the unnecessary check so that Zen Mode toggles off when a user leaves Fullscreen Mode.

To test these changes:
1. Start Virtual Studio Code
2. Enter Fullscreen Mode
3. Enable Zen Mode
4. Exit Fullscreen Mode

Zen mode should now be toggled off.
